src/features/query-bar/flowTypes.js

```diff
@@ -24,6 +24,7 @@ export type ColumnType = {
 export type ConditionType = {
     columnDisplayText: string,
     columnKey: string | null,
+    fieldId: string,
```

Comment:
What's the purpose of this?

Reply:
I need fieldId in `getColumnOptions` in  Condition.js in order to find the corresponding column 

```
const getColumnOptions = () => {
        const { fieldId } = condition;
        const column = columns && columns.find(c => c.id === fieldId);
        if (column && column.options) {
            return column.options.map(option => {
                const { key } = option;
                return {
                    displayName: key,
                    id: fieldId,
                    type: 'enum',
                    value: key,
                };
            });
        }
        return [];
    };
```

Reply:
Should ColumnType.options be Array<OptionType>? It's currently Array<Object>.

```c => c.id === fieldId```. Why does ColumnType.id map to ConditionType.fieldId? Why is the right side called fieldId? Is ConditionType.selectedColumnId a more appropriate name?

Alternatively, you already have ConditionType.columnKey. Is that not sufficient to lookup the Column? I'd prefer that we enable this instead stuffing in another property.

Reply:
So ColumnType does not have a property called key, it has id, so I'm going to use columnId as a property instead of fieldId.

Reply:
I will also be removing columnKey and replacing it with columnId.